lib/RsaSignature2018.js

```diff
@@ -1,6 +1,36 @@
 /*!
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
-// import * as env from './env.js';
+import {JwsLinkedDataSignature} from '@digitalbazaar/jws-linked-data-signature';
+import {
+  RsaVerificationKey2018
+} from '@digitalbazaar/rsa-verification-key-2018';
 
-export class RsaSignature2018 {}
+export class RsaSignature2018 extends JwsLinkedDataSignature {
+  /**
+   * @param type {string} Provided by subclass.
+   *
+   * @param [verificationMethod] {string} A key id URL to the paired public key.
+   *
+   * This parameter is required for signing:
+   *
+   * @param [signer] {function} an optional signer.
+   *
+   * Advanced optional parameters and overrides:
+   *
+   * @param [proof] {object} a JSON-LD document with options to use for
+   *   the `proof` node (e.g. any other custom fields can be provided here
+   *   using a context different from security-v2).
+   * @param [date] {string|Date} signing date to use if not passed.
+   * @param [key] {LDKeyPair} an optional crypto-ld KeyPair.
+   * @param [useNativeCanonize] {boolean} true to use a native canonize
+   *   algorithm.
+   */
+  constructor({
+    signer, key, verificationMethod, proof, date, useNativeCanonize} = {}) {
+    super({type: 'RsaSignature2018', alg: 'PS256',
```

Comment:
The RsaSignature2018 spec says that it uses [RS256](https://datatracker.ietf.org/doc/html/rfc7518#section-3.3) (using PKCS#1 v1.5 padding as opposed to RSA-PSS as used in PS256). While this is an older (though often more compatible) padding scheme, it is the one that was specified for this suite.

Reply:
https://w3c-ccg.github.io/lds-rsa2018/

Reply:
I can help you make the necessary changes and provide a PR if you want.